spec.html

```diff
@@ -32385,6 +32519,156 @@ <h1>Expanded Years</h1>
+        </p>
+
+        <emu-table id="table-time-zone-offset-string-code-points" caption="Time Zone Offset String Code Points">
+          <table>
+            <tr>
+              <th>
+                Code Point
+              </th>
+              <th>
+                Unicode Name
+              </th>
+              <th>
+                Abbreviation
+              </th>
+            </tr>
+            <tr>
+              <td>
+                `U+2212`
+              </td>
+              <td>
+                MINUS SIGN
+              </td>
+              <td>
+                &lt;Minus Sign&gt;
+              </td>
+            </tr>
+          </table>
+        </emu-table>
+
+        <emu-grammar type="definition">
+          UTCOffset :::
+            TemporalSign Hour
+            TemporalSign Hour HourSubcomponents[+Extended]
+            TemporalSign Hour HourSubcomponents[~Extended]
+
+          TemporalSign :::
+            ASCIISign
+            &lt;Minus Sign&gt;
+
+          ASCIISign ::: one of
+            `+` `-`
+
+          Hour :::
+            `0` DecimalDigit
+            `1` DecimalDigit
+            `20`
+            `21`
+            `22`
+            `23`
+
+          HourSubcomponents[Extended] :::
+            TimeSeparator[?Extended] MinuteSecond
+            TimeSeparator[?Extended] MinuteSecond TimeSeparator[?Extended] MinuteSecond TemporalDecimalFraction?
+
+          TimeSeparator[Extended] :::
+            [+Extended] `:`
+            [~Extended] [empty]
+
+          MinuteSecond :::
+            `0` DecimalDigit
+            `1` DecimalDigit
+            `2` DecimalDigit
+            `3` DecimalDigit
+            `4` DecimalDigit
+            `5` DecimalDigit
+
+          TemporalDecimalFraction :::
+            TemporalDecimalSeparator DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit
+            TemporalDecimalSeparator DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit DecimalDigit
+
+          TemporalDecimalSeparator ::: one of
+            `.` `,`
+        </emu-grammar>
+
+        <emu-clause id="sec-istimezoneoffsetstring" type="abstract operation">
+          <h1>
+            IsTimeZoneOffsetString (
+              _string_: a String,
```

Comment:
```suggestion
              _offsetString_: a String,
```
because that's how the algorithm refers to it.